heartbeat/monitors/active/http/config.go

```diff
@@ -153,3 +156,27 @@ func (c *compressionConfig) Validate() error {
 
 	return nil
 }
+
+// Validate validates of the Config object is valid or not
+func (c *Config) Validate() error {
+	if len(c.Hosts) == 0 && len(c.URLs) == 0 {
+		return fmt.Errorf("hosts is a mandatory parameter")
+	}
+
+	if len(c.URLs) != 0 {
+		c.Hosts = append(c.Hosts, c.URLs...)
+	}
+
+	for i := 0; i < len(c.Hosts); i ++ {
```

Comment:
We should break this out to a separate function, something like `normalizeHostURL(str string)` and add unit tests for it.